test/smoke/FirefoxPrivateVPNUITest/FirefoxPrivateVPNUITest/Screens/LanguageScreen.cs

```diff
@@ -0,0 +1,124 @@
+            this.backButton = languageView.FindElementByName("Back");
+            this.title = languageView.FindElementByName("Language");
+            var languagePane = languageView.FindElementByClassName("ScrollViewer");
+            this.defaultLanguageTitle = languagePane.FindElementByName("Default");
+            var defaultLanguagesElements = languagePane.FindElementsByName("English (United States)");
+            this.defaultLanguageRadioButton = defaultLanguagesElements[0];
+            this.defaultLanguageDetails = defaultLanguagesElements[1];
+            this.additionalLanguagesListTitle = languagePane.FindElementByName("Additional");
+            this.additionalLanguagesListView = languagePane.FindElementByAccessibilityId("SupportedLanguagesList");
+        }
+
+        /// <summary>
+        /// Get title on Language screen.
+        /// </summary>
+        /// <returns>The title string.</returns>
+        public string GetTitle()
+        {
+            return this.title.Text;
+        }
+
+        /// <summary>
+        /// Get the text on default language pane.
+        /// </summary>
+        /// <returns>The text on default language pane.</returns>
+        public string GetDefaultLanguageTitle()
+        {
+            return this.defaultLanguageTitle.Text;
+        }
+
+        /// <summary>
+        /// Get the text on additional language pane.
+        /// </summary>
+        /// <returns>The text on additional language pane.</returns>
+        public string GetAdditionalLanguagesPaneTitle()
+        {
+            return this.additionalLanguagesListTitle.Text;
+        }
+
+        /// <summary>
+        /// Get the text on default language radio button.
+        /// </summary>
+        /// <returns>The text on default language radio button.</returns>
+        public string GetTextOnDefaultLanguageRadioButton()
+        {
+            return this.defaultLanguageRadioButton.Text;
+        }
+
+        /// <summary>
+        /// Get the details on default language radio button.
+        /// </summary>
+        /// <returns>The details on default language radio button.</returns>
+        public string GetDetailsOnDefaultLanguageRadioButton()
+        {
+            return this.defaultLanguageDetails.Text;
+        }
+
+        /// <summary>
+        /// Click the Back button.
+        /// </summary>
+        public void ClickBackButton()
+        {
+            this.backButton.Click();
+        }
+
+        /// <summary>
+        /// Is default language radio button checked.
+        /// </summary>
+        /// <returns>Default language radio button checked or not.</returns>
+        public bool IsDefaultLanguageRadioButtonChecked()
+        {
+            return this.defaultLanguageRadioButton.Selected;
+        }
+
+        /// <summary>
+        /// Random pick a language from list.
+        /// </summary>
+        public void RandomPickAdditionalLanguage()
```

Comment:
Careful with this, as many strings are hardcoded here in English.

Reply:
It is ok for now. We reset to English after picking another language.